homework-week3.js

```diff
@@ -0,0 +1,196 @@
+// 1. Strings!
+
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);   //hello,this,is,a,difficult,to,read,sentence
+console.log(myString.length);   //42
+
+//Find a way to remove the commas from the string and replace them with spaces.
+for (let i=0; i<myString.length; i++){
+    myString = myString.replace(/,/g, " "); 
+    console.log(myString);   //"hello this is a difficult to read sentence"
+}
+
+//2. Arrays!
+
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+//Add a statement that adds Mauro's favorite animal 'turtle' to the existing array.
+favoriteAnimals.push("turtle");
+
+console.log(favoriteAnimals);    //(4) ["blowfish", "capricorn", "giraffe", "turtle"]
+
+//add 'meerkat', but make sure it will be placed after 'blowfish' and before 'capricorn'.
+favoriteAnimals.splice(1, 0, "meerkat");  // 1: the index of the new element,    0: the number of the elements I want to remove  
+
+console.log("This statement should add the new element 'meerkat' in the second position which has the index 1 without removing any other elements from the existing array.");
+console.log(favoriteAnimals);   //(5) ["blowfish", "meerkat", "capricorn", "giraffe", "turtle"]
+
+//Log the length of the array
+console.log(favoriteAnimals.length);
+console.log("The array has a length of: " + favoriteAnimals.length);   //The array has a length of: 5
+
+// delete 'giraffe' from the array.
+const removed = favoriteAnimals.splice(3, 1);   // I'll delete more elements => I'll use let to declare my variable
+/* 3: the index of the element we want to remove,    
+1: how many times we want to repeat the deletion of elements which will have the index 3 after the first deletion 
+*/
+console.log(favoriteAnimals);    //(4) ["blowfish", "meerkat", "capricorn", "turtle"]
+
+// you want to delete 'meerkat', but you don't know the position or the index of the item in the array, how can you find it?
+favoriteAnimals.indexOf("meerkat");     // we find the index of the element ---- it is : 1
+console.log("The item you are looking for is at index: " + favoriteAnimals.indexOf("meerkat"));
+//removed = favoriteAnimals.splice(1, 1);  // we delete it using .splice(the index, how many times)
+//console.log(favoriteAnimals);
+
+
+// ------ More JavaScript --------
+
+// 3 arguments function that returns the sum
+function sum (a,b,c) {
+    return a+b+c;
+}
+sum (1,2,3);   // 6
+
+//---------colorCar function -----------
+function colorCar (color) {
+    return "a "+ color + " car";   //the input should be a string .... I tried a lot but couldn't use red as a color not as a string????
```

Comment:
The function works as it should, it returns 'a red car', do you still have a question about this task?